src/valkey-benchmark.c

```diff
@@ -1557,6 +1523,17 @@ int parseOptions(int argc, char **argv) {
         "                    If the command is supplied on the command line in cluster\n"
         "                    mode, the key must contain \"{tag}\". Otherwise, the\n"
         "                    command will not be sent to the right cluster node.\n"
+        " --rfr <mode>       Enable read from replicas in cluster mode.\n"
+        "                    This command must be used with the --cluster option.\n"
+        "                    When using this option, it is recommended to use only \n"
+        "                    the commands for read requests.\n"
+        "                    There are two mode for reading from replicas.\n"
+        "                    If nothing is entered or 'replicas' is entered, this\n"
+        "                    sends read requests to replicas only (default) \n"
+        "                    If 'all' is entered, this sends read requests to all nodes,\n"
+        "                    including primaries and replicas. \n"
+        "                    If you only want to request the primaries, use --cluster option only.\n"
+        "                    2=read from all nodes(primary and replica)\n"
```

Comment:
```suggestion
        " --rfr <mode>       Enable read from replicas in cluster mode.\n"
        "                    This command must be used with the --cluster option.\n"
        "                    There are three modes for reading from replicas:\n"
        "                    'no' - sends read requests to primaries only (default) \n"
        "                    'yes' - sends read requests to replicas only.\n"
        "                    'all' - sends read requests to all nodes.\n"
        "                    Since write commands will not be accepted by replicas,\n"
        "                    it is recommended to enable read from replicas only for read command tests.\n"
```